build-system-tests/scripts/mega-app-copy-files.sh

```diff
@@ -152,6 +152,12 @@ if [[ "$FRAMEWORK" == 'angular' ]]; then
         echo "npx strip-json-comments mega-apps/${MEGA_APP_NAME}/tsconfig.app.json | npx json -a -e 'this.files.push(\"src/polyfills.ts\")' >tsconfig.app.json.tmp && mv tsconfig.app.json.tmp ./mega-apps/$MEGA_APP_NAME/tsconfig.app.json && rm -f tsconfig.app.json.tmp"
         npx strip-json-comments mega-apps/${MEGA_APP_NAME}/tsconfig.app.json | npx json -a -e 'this.files.push("src/polyfills.ts")' >tsconfig.app.json.tmp && mv tsconfig.app.json.tmp ./mega-apps/$MEGA_APP_NAME/tsconfig.app.json && rm -f tsconfig.app.json.tmp
     fi
+    if [[ "$FRAMEWORK_VERSION" == 14 ]]; then
```

Comment:
Might make sense to add a comment explaining the reasoning for the special handling of Angular 14 here?

Reply:
Yup makes sense, added